app/src/lib/menu-update.ts

```diff
@@ -80,7 +80,50 @@ function menuItemStateEqual(state: IMenuItemState, menuItem: MenuItem) {
   return true
 }
 
+const allMenuIds: ReadonlyArray<MenuIDs> = [
```

Comment:
This is probably the only janky part of the change - I'd love to keep this in sync with the `MenuIDs` source but I won't hold up this PR unless someone has some bright ideas to solve this with TYPES!

Reply:
Yeah, less than ideal but I don't know of a better way 🤔